server/data/resolvers.js

```diff
@@ -52,19 +51,16 @@ const resolvers = {
     tasks: (user) => Task.find({claimer: user.id}).then(tasks => tasks),
     tokenChanges: (user) => Token.find({userId: user.id}).then(tokens => tokens),
     validations: (user) => Validation.find({userId: user.id}).then(validations => validations),
-    votes: (user) => Vote.find({userId: user.id}).then(votes => votes),
-    voteRecords: (user) => User.findOne({_id: user.id}).then(userDoc => userDoc.voteRecords)
-  },
-  VoteRecord: {
-    voter: (vote) => User.findOne({account: vote.voter}).then(user => user),
-    task: (vote) => Task.findById(vote.task).then(voteRecord => voteRecord)
+    votes: (user) => Vote.find({voter: user.id}).then(votes => votes)
```

Comment:
Fun fact, you don't need the code after then `then` unless it's tranforming the result in some way.  You can just write `Vote.find({voter: user.id})`